BLUESPAWN-client/BLUESPAWN-client.vcxproj

```diff
@@ -302,7 +329,55 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>Secur32.lib;DbgHelp.lib;Wintrust.lib;ws2_32.lib;Crypt32.lib;Shlwapi.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <EnableUAC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">

```

Comment:
Remove the condition statement and make this more concise

Reply:
deleted entirely as the full manifest is specified in the file and not generated